CONTRIBUTING.md

```diff
@@ -413,15 +413,28 @@ rule is that they should be as concise as possible.  Specifically:
 * use variable annotations instead of type comments, even for stubs
   that target older versions of Python.
 
-Stub files should only contain information necessary for the type
-checker, and leave out unnecessary detail:
-* for arguments with a default, use `...` instead of the actual
-  default;
-* for arguments that default to `None`, use `Foo | None` explicitly
-  (see below for details);
-* use `float` instead of `int | float`.
+The primary users for stub files are type checkers,
+so stub files should generally only contain information necessary for the type
+checker, and leave out unnecessary detail.
+However, stubs also have other use cases:
+* stub files are often used as a data source for IDEs,
+  which will often use the signature in a stub to provide information
+  on functions or classes in tooltip messages.
+* stubs can serve as useful documentation to human readers,
+  as well as machine-readable sources of data.
+
+As such, typeshed's policy on some issues of style has been revised.
+Whereas typeshed used to recommend omitting default values
+for function and method parameters,
+we now recommend that default values be retained for "simple" default values
+(e.g. bools, ints, bytes, strings, and floats are all now permitted).
```

Comment:
I'd revise this to just document the status quo. Historical information is not really useful to the reader.